sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/util/Transforms.java

```diff
@@ -510,6 +501,14 @@ private static DocumentKeyValueElement toDocumentKeyValueElement(
         return documentKeyValueElement;
     }
 
+    private static Map<String, DocumentFieldSchema> toDocumentFieldProperties(
+        Map<String, com.azure.ai.formrecognizer.documentanalysis.implementation.models.DocumentFieldSchema> properties) {
```

Comment:
NIT: it could be easy to use customized autogene to share one single DocumentFieldSchema?

Reply:
I will try some customization in a follow up PR.